### PR TITLE
sub-PR 6

### PR DESCRIPTION
**Pull request by drichte2**
Last one! Nearly at a state where we'll finalize the ASD namelist file

### CODE REVIEW

File: src/droplet.F [hunk 1]
```diff
@@ -413,6 +415,12 @@ subroutine droplet_driver(dt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,
                                  xh,xf,yh,yf,zh,zf,zs,znt,sigma,sigmaf, &
                                  ua,va,wa,ta,qa,rho,prs,sigdot)
 
+      !Temporary for testing purposes:
+      rhval = (ru*tval*rhoval*qval)/(mw*eslf(prsval,tval))
+      if (rhval .lt. 0.3 .or. rhval .gt. 1.2) then
+      write(*,'(a8,i,8e15.6)') 'DHR3',np,rhval,tval,rhoval,qval,x3d,y3d,z3d,pdata(np,prrp)
+      end if
```
Comment (sjsprecious) [inline src/droplet.F]: Do we have to keep this `write` statement? The `write` function is not supported on the GPU.

Comment (drichte2) [reply src/droplet.F]: Could you just comment it out for now? We won't need it in the long run, but it's been helpful for setting up the case

Comment (sjsprecious) [reply src/droplet.F]: Ok, that sounds good. I will just comment it out for now.

**Comment by sjsprecious** [comment]
By using the namelist `namelist.restC` and without the conda environment, the verification result on Gust looks below:

---

CPU (intel, 2 MPI ranks) vs. GPU (nvhpc, 2 MPI ranks + 2 A100 GPUs)

Stat output comparison

Identical variables: 59
variables with rel_diff > 1.e-6: 23
variables with rel_diff <= 1.e-6: 4
four variables with largest error: PPMIN, PPMAX, VOR2KM, VOR1KM

Droplet 0th-order diagnostic comparison

Identical variables: 10
variable with rel_diff > 1.e-6: 2
variable with rel_diff <= 1.e-6: 0
three variables with largest error: radavg (~3.7e-6), prsavg (~1.1e-6)

Droplet 1st-order diagnostic comparison

Identical variables: 512
variable with rel_diff > 1.e-6: 0
variable with rel_diff <= 1.e-6: 0

---

Compared to the PR #108, the stat output comparison result shows one more variable with relative difference >1.e-6 and four variables with relative difference <1.e-6 (which are **THPMAX**, **STHPMN**, **SWSPMX** and **KSHMAX**). In the PR #108. the five variables with relative difference <1.e-6 are **THPMAX**, **THPMIN**, **RHMIN**, **TKEMAX** and **STHPMX**. The 0th-order droplet diagnostic shows two more identical variables and the variables with relative difference >1.e-6 are now **radavg** and **prsavg**. The 1st-order droplet diagnostic is consistent. The verification results are generally improved compared to PR #108.

**Review by johnmauff — APPROVED** [review]
If @sjsprecious is fine with this PR I am fine.

**Review by sjsprecious — APPROVED** [review]

**Pull request merged**